src/components/volunteer/VolunteerSteps.jsx

```diff
@@ -0,0 +1,374 @@
+          <ol>
+            <li>
+              Read our{' '}
+              <a
+                href="https://github.com/codeforpdx/codeofconduct"
+                target="_blank"
+                rel="noreferrer"
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                Code of Conduct
+              </a>
+            </li>
+            <li>
+              Complete our{' '}
+              <a
+                href="https://docs.google.com/forms/d/e/1FAIpQLSfu4u9FB00RXd2TncbMLws_Ckot-vYqPK0oUHr81uLDjThE7Q/viewform"
+                target="_blank"
+                rel="noreferrer"
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                onboarding form
+              </a>
+            </li>
+            <li>
+              Review our{' '}
+              <a
+                href="https://github.com/codeforpdx"
+                target="_blank"
+                rel="noreferrer"
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                current projects
+              </a>
+            </li>
+            <li>
+              Read the{' '}
+              <a
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                ADA Compliance Guide
+              </a>
+              , all CODE PDX projects are inclusive by design
+            </li>
+            <li>
+              Join a project demo night & onboarding session listed on{' '}
+              <a
+                href="https://www.meetup.com/code-for-pdx/"
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                meetup
+              </a>{' '}
+              and the{' '}
+              <a
+                href="/#events"
+                style={{
+                  color: theme.palette.primary.main,
+                  textDecoration: 'none',
+                  fontWeight: 600
+                }}
+              >
+                events calendar
+              </a>
+            </li>
+          </ol>
+        </Step> */}
+
+        {/* ========== START HERE */}
+
+        <Step step={1} title="Self-Onboarding & Initial Contact" mb={36.25}>
+          <Typography variant="h6">Get familiar with CODE PDX</Typography>
+          <List dense>
+            <ListItem>
+              <ListItemText>
+                1. Read our{' '}
```

Comment:
You've already mentioned needing to clean up the code, but I'll note a few things for the record and in case it helps anyone just reading through this.

There are a lot of `{' '}` that can all be nixed